src/screens/Home/Tabs/workspacesController.js

```diff
@@ -0,0 +1,199 @@
+const ipcRendererManager = require("../../../utils/ipcRendererManager");
+const Toast = require("../../../components/Toast/toast.js");
+
+function setFavorite(favoriteCell) {
+  if (favoriteCell.classList.contains("favorite")) {
+    favoriteCell.classList.remove("favorite");
+    favoriteCell.classList.add("not-favorite");
+    favoriteCell.setAttribute("data-tooltip", "Add to favorites");
+  } else {
+    favoriteCell.classList.remove("not-favorite");
+    favoriteCell.classList.add("favorite");
+    favoriteCell.setAttribute("data-tooltip", "Remove from favorites");
```

Comment:
You can use classList.toggle instead of using if to decide between .remove and .add